drivers/usb/uhc/Kconfig

```diff
@@ -3,7 +3,7 @@
 
 
 menuconfig UHC_DRIVER
-	bool "USB host controller driver [EXPERIMENTAL]"
+	bool "USB host controller [EXPERIMENTAL]"
```

Comment:
bool "USB host controller drivers [EXPERIMENTAL]"

Reply:
Fixed, thank you.